reinforcement_learning/rlclientlib/logger/eventhub_client.h

```diff
@@ -35,11 +67,7 @@ namespace reinforcement_learning {
     std::string _authorization;
     long long _authorization_valid_until; //in seconds
     std::mutex _mutex;
-
-    // cannot be copied or assigned
-    eventhub_client(const eventhub_client&) = delete;
-    eventhub_client(eventhub_client&&) = delete;
-    eventhub_client& operator=(const eventhub_client&) = delete;
-    eventhub_client& operator=(eventhub_client&&) = delete;
+    moving_queue<task> _tasks;
+    const size_t _tasks_count;
```

Comment:
The naming is confusing. This reads like a cache of "_tasks.size()". Consider renaming "_max_task_count" or something similar.